controllers/user/UserController.php

```diff
@@ -2,6 +2,8 @@
 
 namespace humhub\modules\smartVillage\controllers\user;
 
+use humhub\modules\legal\models\Page;
+use humhub\modules\legal\models\RegistrationChecks;
```

Comment:
can we add a required dependency on the "legal" module? or do we not need it? what happen if we try to register if the legal module is not activated for example?

Reply:
Hi @donni106 
Yes, we can add required dependency on the legal module and or we can add condition if the legal module is not present or disabled then it will skip the privacy policy code.

As of now, if the legal module is not activated then it will throw an error so, we have to handle that error.

Reply:
Hi @donni106 
We pushed a commit where we are checking legal module is activated or not and if it is not activated then privacy policy code or function will not executed.
Thanks!

Reply:
good decision!